.github/workflows/win-wheels.yaml

```diff
@@ -43,15 +44,15 @@ jobs:
           cat setup.cfg
 
       - name: Cache vcpkg
-        uses: actions/cache@v1
+        uses: actions/cache@v3
         id: vcpkgcache
         with:
           path: C:/vcpkg/installed
-          key: ${{ runner.os }}-vcpkg-${{ hashFiles('.github/workflows/*') }}
+          key: ${{ runner.os }}-dependencies-${{ hashFiles('.github/workflows/*', 'vcpkg.json') }}
 
       - name: Install GDAL
         run: |
-          vcpkg install gdal:x64-windows
+          vcpkg install --feature-flags="versions" gdal:x64-windows
```

Comment:
So `vcpk install gdal` is also what I did originally in the pyogrio wheel builds, but when we moved to use a manifest file (to pin the version, or provide overrides), some more things changed:

- `gdal` is now listed as package to install in the manifest, so you don't have to (or can't?) mention it here, I think
- We are also using `--feature-flags="manifests"` in addition to "versions"
- We also pointed to the directory where the manifest file is living with `--x-manifest-root=./ci` (although this probably not needed here since here it lives top-level)

See the full diff of https://github.com/geopandas/pyogrio/pull/69

Reply:
@jorisvandenbossche thank you! I'm going to try removing `gdal:x64-windows` from the command line. Maybe it's negating the manifest? Hard to say from the vcpkg docs.